src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -48,18 +50,15 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
   private def handleStart(): Unit = {
     // Record the original sender here and not in the yield over the `Future`, as the
     // value of sender() when that code executes may be different than what it is here.
-    log.info("In handleStart for " + call.fullyQualifiedName)
     val originalSender = sender()
     val backendInputs = backend.adjustInputPaths(call, locallyQualifiedInputs)
-    log.info("Got backend inputs for " + call.fullyQualifiedName + ": " + backendInputs)
 
     val tryCommand = for {
       commandLine <- call.instantiateCommandLine(backendInputs)
     } yield {
-      log.info("Command line for " + call.fullyQualifiedName + ": " + commandLine)
+      log.info(s"$tag: launching `$commandLine`")
```

Comment:
addendum to my last statement, or perhaps just a match on `call.instantiateCommandLine(backendInputs)`, which each calls the appropriate handler function